backend/api-documentation/components/schemas.js

```diff
@@ -1,13 +1,63 @@
 module.exports = {
-  TestData: {
-    type: 'object',
-    additionalProperties: false,
-    required: ['data'],
-    properties: {
-      data: {
-        type: 'string',
-        example: 'This is a test string',
-      },
-    },
-  },
+	TestData: {
+		type: 'object',
+		additionalProperties: false,
+		required: ['data'],
+		properties: {
+			data: {
+				type: 'string',
+				example: 'This is a test string',
+			},
+		},
+	},
+	SocialSecurityNumber: {
```

Comment:
Shouldn't it be `PersonalNumber` or something similar since it maps to `pnr` in the db? Just for clarity.